test/common/upstream/load_balancer_fuzz_base.cc

```diff
@@ -0,0 +1,241 @@
+#include "test/common/upstream/load_balancer_fuzz_base.h"
+
+#include "test/common/upstream/utility.h"
+
+#define MAX_NUM_HOSTS_IN_PRIORITY_LEVEL 256
+
+namespace Envoy {
+namespace Upstream {
+
+void LoadBalancerFuzzBase::initializeASingleHostSet(
+    const test::common::upstream::SetupPriorityLevel& setup_priority_level,
+    const uint8_t priority_level) {
+  uint32_t num_hosts_in_priority_level = setup_priority_level.num_hosts_in_priority_level();
```

Comment:
Nit: `const` everything that can be made const.

Reply:
Added const clause to every field that is constant.